src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,14 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        } else {
+            return user.getPassword().equals(password);
+        }
```

Comment:
try to rewrite this without if-else, and `false` literal, return boolean expression directly